packages/engine-server/src/markdown/utilsv5.ts

```diff
@@ -431,7 +417,7 @@ export class MDUtilsV5 {
     });
 
     // add additional plugin for publishing
-
+    //maybe something here is interfering
```

Comment:
Is this left over from debugging, or is there still an issue here?

Reply:
good catch - it's leftover from debugging, will remove.